pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -688,6 +688,13 @@ func buildSidecarOutboundTCPListenerOptsForPortOrUDS(listenerMapKey *string,
 	var destinationCIDR string
 	if len(listenerOpts.bind) == 0 {
 		svcListenAddress := listenerOpts.service.GetAddressForProxy(listenerOpts.proxy)
+		// if listenerOpts.service is not the common k8s svc and headless svc, it may be a svc converted from service entry, etc.
+		// It's hard to detect IP family for this kind of service, such se with only host name, so follows the current proxy's
```

Comment:
// It's hard to detect IP family for this kind of service, such se with only host name, so follows the current proxy's

This is unclear to me

Reply:
Hi @ramaraochavali,  it's okay for normal k8s service in the service mesh, but if there is a service entry out of service mesh, even out of k8s cluster, then we can not really know its service ip family is IPv4, IPv6 or both. So that's what I want to explain.

Reply:
Suggest to change
"It is detect IP family for service entries, so use the current proxy's IP Mode."

Reply:
Okay, thanks @ramaraochavali

Reply:
done